src/test/resources/bodies/search/tools.json

```diff
@@ -3,12 +3,12 @@
  "_source": false,
  "query": {
   "match": {
-   "_type": "tool"
+   "_index": "workflows"
```

Comment:
why did the `tools.json` change to index workflows?

Reply:
Not sure; I'm not even using this file for now. Reverted back in e5e32a57147b623431baaac560ce837c1ea3df21